### DISCUSSION TITLE
config.js prConcurrentLimit not working in combination with "extends": ["config:base"]?

### DISCUSSION THREAD

**Discussion by japj** [renovatebot/renovate#8624] [Request Help] [answered]
(I thought I already submitted this question, but I can't find it anymore, 2nd try)

I am running on self-hosted bot (with a config.js) targeting a git repository with a renovat.json file.
I'm using "extends": ["config:base"] and am trying to 'overrule' the prConcurrentLimit but it seems that it still tries for 20.

relevant config.js content:
```
module.exports = {
    extends: ["config:base"],
    ...
    prConcurrentLimit: 5,
    ...
}
```

(Combined config) content:
```
"config": {
    "description": [
        "Limit to maximum 20 open PRs at any time",
        ....
    ],
    ...
    "prConcurrentLimit": 5,
    ...
}
```
relevant renovate.json content:
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ]
}
```
relevant bot logging:
```
DEBUG: Calculating prConcurrentLimit (20) (repository=....)
```

**Comment by viceice** [comment, marked as answer]
You need to configure it in `force` block, if you like to have it from bot, otherwise the repo config is overriding the bot config

```js
module.exports = {
    extends: ["config:base"],
    ...
    force: {
        prConcurrentLimit: 5,
    }
    ...
}
```

**Reply by viceice** [reply]
https://docs.renovatebot.com/self-hosted-configuration/#force

**Reply by japj** [reply]
thanks, that solved it
